.azure_pipelines/job_templates/olive-example-template.yaml

```diff
@@ -43,6 +43,10 @@ jobs:
       displayName: Set exampleRequirements
       condition: eq(variables['exampleRequirements'], '')
 
+    - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+      displayName: Add conda to PATH
+      condition: and(eq(parameters.name, 'Windows_CI'), eq(variables['exampleFolder'], 'mobilenet_qnn_qualcomm_npu'))
```

Comment:
>Windows_CI [](http://example.com/codeflow?start=42&length=10)

there is a concept named "windows" in test template. do we need introduce to example to keep consistent?

Reply:
Updated.